server/routes/index.ts

```diff
@@ -1,23 +1,23 @@
 // Any route that comes in, send it to the universalLoader
 
-import express from 'express'
+import express, { RequestHandler } from 'express'
 import { createStore, universalLoader } from '../universal'
 import { appServer } from '../../src/paths'
 import { resolveModule } from '../../src/util'
+import { isReservedPath } from '../helpers/is-reserved-path.helper'
 
 const router = express.Router()
 
 const preLoadState = resolveModule(appServer + '/preLoadState')
-const middleware =
-  function(req: { baseUrl: string | string[] }, res: any, next: () => void) {
-    if(preLoadState)
-      if(req.baseUrl.indexOf('.') !== -1 || req.baseUrl.indexOf('api') !== -1 || req.baseUrl.indexOf('static') !== -1)
```

Comment:
When this occurs?
```js
req.baseUrl.indexOf('.') !== -1
```

Reply:
when route path is assets, css o js file

Reply:
fixed